src/blockchain/blockchain.service.ts

```diff
@@ -25,6 +25,13 @@ export class BlockchainService {
     return JSON.stringify(passphrase.Mnemonic.generateMnemonic());
   }
 
+  public async getFaucetTokens(address: string): Promise<void> {
+    await this.client.invoke('faucet:fundTokens', {
+      address,
+      tokens: 500
+    });
+  }
```

Comment:
As we discussed, we can use `token:transfer` transaction here instead of faucet. As it's our backend we can set from which account to transfer and sign the transaction.